a10_neutron_lbaas/a10_context.py

```diff
@@ -108,6 +111,19 @@ def __exit__(self, exc_type, exc_value, traceback):
         super(A10WriteContext, self).__exit__(exc_type, exc_value, traceback)
 
 
+class A10ReplayContext(A10WriteContext):
+
+    def __init__(self, *args, **kwargs):
+        self._appliance = kwargs.pop('appliance', None)
```

Comment:
vThunder?  I stick to generic names like "appliance" and "device" because the trade names will inevitably change. "Appliance" is my preferred nomenclature because regardless of whether it's a hardware or software vThunder/yet-to-exist-product, it's a thing that runs an application.